src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.cpp

```diff
@@ -29,12 +29,20 @@
 
 namespace CurvedScalarWave::Worldtube::OptionTags {
 SelfForceOptions::SelfForceOptions() = default;
-SelfForceOptions::SelfForceOptions(double mass_in, size_t iterations_in)
-    : mass(mass_in), iterations(iterations_in) {}
+SelfForceOptions::SelfForceOptions(const double mass_in,
+                                   const size_t iterations_in,
+                                   const double turn_on_time_in,
+                                   const double turn_on_interval_in)
+    : mass(mass_in),
+      iterations(iterations_in),
+      turn_on_time(turn_on_time_in),
+      turn_on_interval(turn_on_interval_in) {}
```

Comment:
Is it ok if the turn on interval is zero? I believe it's in the denominator of the exponent

Reply:
Yes, good catch. I limited it to 1M now which is still very low but can not cause any numerical issues.